python/lab4-propprices.md

```diff
@@ -18,17 +18,17 @@ ___
 
 #### Task 2: Importing the Data
 
-In a previous lab, we used the Python `csv` module to parse CSV files. However, because we're working with structured data, the Spark SQL framework can be easier to use and provide better performance. We are going to use the `pyspark_csv` third-party open source module to create a `DataFrame` from an RDD of CSV lines.
+In a previous lab, we used the Python `csv` module to parse CSV files. However, because we're working with structured data, the Spark SQL framework can be easier to use and provide better performance. We are going to use the `spark.read.csv` to create a `DataFrame` from an RDD of CSV lines.
 
-> **NOTE**: The `pyspark_csv.py` file is in the `~/externals` directory on the appliance. You can also [download it yourself](https://github.com/seahboonsiew/pyspark-csv) and place it in some directory.
+> **NOTE**: The `pyspark_csv.py` file, a third-party package for loading csv files for Spark 1.x, is in the `~/externals` directory on the appliance. You can also [download it yourself](https://github.com/seahboonsiew/pyspark-csv) and place it in some directory.
 > 
 > This module also depends on the `dateutils` module, which typically doesn't ship with Python. It is already installed in the appliance. To install it on your own machine, run the following from a terminal window:
 
 ```
 sudo easy_install dateutils
 ```
 
-To import `pyspark_csv`, you'll need the following snippet of code that adds its path to the module search path, and adds it to the Spark executors so they can find it as well:
+To import `pyspark_csv` (if needed in Spark 1.x, for Spark 2.x, skip this step), you'll need the following snippet of code that adds its path to the module search path, and adds it to the Spark executors so they can find it as well:
```

Comment:
I'd rephrase to something like this:

"If you're using Spark 1.x, you will need the following snippet to import the `pyspark_csv` module. In Spark 2.0, the built-in `spark.read.csv` method is sufficient, so this step is not needed."